src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Edit.cshtml

```diff
@@ -28,6 +28,29 @@ else
 
 <div asp-validation-summary="ModelOnly"></div>
 <form asp-action="Edit" asp-route-returnUrl="@ViewData["returnUrl"]" method="post">
+    <div class="card mb-3 position-sticky action-bar">
+        <div class="card-body bg-primary p-3 text-right">
```

Comment:
```suggestion
        <div class="card-body bg-primary p-2 text-right">
```

Reply:
Please add the same thing for the create view.

Reply:
Ok